package/python-kiwi-spec-template

```diff
@@ -127,27 +127,39 @@ This metapackage installs the necessary system dependencies
 to run KIWI.
 
 %package -n kiwi-systemdeps-containers
-Summary:        KIWI - host requirements for container images
+Summary:        KIWI - host requirements for OCI container images
 Group:          %{sysgroup}
 Provides:       kiwi-image-docker-requires = %{version}-%{release}
 Obsoletes:      kiwi-image-docker-requires < %{version}-%{release}
-Provides:       kiwi-image-wsl-requires = %{version}-%{release}
-Obsoletes:      kiwi-image-wsl-requires < %{version}-%{release}
 %if "%{_vendor}" != "debbuild"
 Provides:       kiwi-image:docker
-Provides:       kiwi-image:appx
 %endif
 %if 0%{?suse_version}
 Requires:       umoci
-Requires:       fb-util-for-appx
 %else
 Requires:       buildah
 %endif
 Requires:       skopeo
 
 %description -n kiwi-systemdeps-containers
 Host setup helper to pull in all packages required/useful on
-the build host to build container images e.g docker, wsl
+the build host to build OCI container images
+
+%package -n kiwi-systemdeps-containers-wsl
+Summary:        KIWI - host requirements for WSL container images
+Group:          %{sysgroup}
+Provides:       kiwi-image-wsl-requires = %{version}-%{release}
+Obsoletes:      kiwi-image-wsl-requires < %{version}-%{release}
+%if "%{_vendor}" != "debbuild"
+Provides:       kiwi-image:appx
+%endif
+%if 0%{?suse_version}
+Requires:       fb-util-for-appx
+%endif
```

Comment:
This is bad. It makes this package pointless on non-SUSE systems.

Reply:
@Conan-Kudo I also wondered about this part, but this is what we had in the spec before. Here is the former ```kiwi-systemdeps-containers``` package spec

```
%package -n kiwi-systemdeps-containers
Summary:        KIWI - host requirements for container images
Group:          %{sysgroup}
Provides:       kiwi-image-docker-requires = %{version}-%{release}
Obsoletes:      kiwi-image-docker-requires < %{version}-%{release}
Provides:       kiwi-image-wsl-requires = %{version}-%{release}
Obsoletes:      kiwi-image-wsl-requires < %{version}-%{release}
%if "%{_vendor}" != "debbuild"
Provides:       kiwi-image:docker
Provides:       kiwi-image:appx
%endif
%if 0%{?suse_version}
Requires:       umoci
Requires:       fb-util-for-appx
%else
Requires:       buildah
%endif
Requires:       skopeo
```

So you see ```fb-util-for-appx``` was only used in combination with suse systems. I guess we had a reason to do it that way ? Probably because it does not exist for others ? We also have WSL integration tests for suse systems building. Which brings me to the question if and how we support WSL containers for other OS'es ?

Reply:
We probably failed to update this when `appx-util` got added to Fedora and EPEL.